examples/readstringlist.cf

```diff
@@ -0,0 +1,39 @@
+#+begin_src prep
+#@ ```
+#@ printf "one\ntwo\nthree" > /tmp/list.txt
+#@ printf "1\n2\n3"         > /tmp/list.txt
+#@ printf "1.0\n2.0\n3.0"   > /tmp/list.txt
+#@ ```
+#+end_src
+bundle agent example_readstringlist
+# @brief Example showing readstringlist()
+{
+  vars:
+      "my_list_of_strings"
+        slist => readstringlist( "/tmp/list.txt", # File to read
+                              "",                 # Ignore any non-digits
+                              "\n",               # Split on newlines
```

Comment:
How is an empty regex `""` ignoring any non-digits where previously a different regex was used with the same comment? `"^(\D+)",         # Ignore any non-digits`

Reply:
bogus comment, thanks.